_i18n/fr/index.md

```diff
@@ -15,7 +15,7 @@
                         <div class="main-info">
                             <h1 id="main-h1">Monero</h1>
                             <h2 id="main-h2">Monnaie Digitale Confidentielle</h2>
-                            <p id="main-text">Monero de l'argent pour un monde connecté. C'est rapide, confidentiel et sécurisé. Avec Monero, vous êtes votre propre banque. Vous pouvez le dépenser en toute sécurité, en sachant que perdonne ne peut voir votre solde ni suivre votre activité.</p>
+                            <p id="main-text">Monero, c'est de l'argent pour un monde connecté. C'est rapide, confidentiel et sécurisé. Avec Monero, vous êtes votre propre banque. Vous pouvez le dépenser en toute sécurité, en sachant que personne ne peut voir votre solde ni suivre votre activité.</p>
```

Comment:
Maybe: Monero est une monnaie pour un monde connecté?

Reply:
ok

Reply:
i was focus on cash in the vo...